botmother/utils/api.py

```diff
@@ -23,12 +23,12 @@ def __init__(self, token=None, chat_id=None):
         if chat_id:
             self.chat_id = chat_id
 
-    def send(self, method, data, files=None):
+    def send(self, method, data={}, files=None):
```

Comment:
Use ‘None’ instead

Reply:
None doesn't work here, I tried that's why use empty objecct.